source/pages/Connections/ConnectWallet.tsx

```diff
@@ -50,9 +50,9 @@ export const ConnectWallet = () => {
     setIsLoading(false);
   };
 
-  useEffect(() => {
-    if (!isActiveNetwork(chain, chainId)) changeNetwork();
-  }, []);
+  // useEffect(() => {
```

Comment:
Was this intentionally committed?
This hook is important for when the wallet is on network X and the connection request is from Y

Reply:
yeah only for testing purposes until we have the bug [dapp - demo changing wallet active network randomly by itself](https://app.clickup.com/t/319tw0m) fixed